### ISSUE TITLE
Meeting ADA Accessibility Requirements across the Web

### ISSUE THREAD

**Issue by AL0YSI0US** [CommonsBuild/coordination#899]
The Community Steward that will support and review this work is : @ 

## 👇🏽  What is this issue about?

As I was falling down the 🐇 . . . . 🕳️ (rabbit hole) that is TEC I noticed there were parts of our sites that won't reach our entire audience. = ♿ 

I ran tests on a  few pages and will include photos in the comments within this issue to demonstrate a quick audit and allow others to see what I am seeing. There are changes that will need to be made to the HTML files for the website and I haven't a clue how edits are done to the handbook 😅

## ✨  What does success look like? (i.e., the criteria to meet to call this issue done)
1.  Accessibility auditing of the TEC Website and Handbook
> [WAVE](https://wave.webaim.org/) is a suite of evaluation tools that helps authors make their web content more accessible to individuals with disabilities. WAVE can identify many accessibility and Web Content Accessibility Guideline (WCAG) errors, but also facilitates human evaluation of web content. ( I ❤️ this tool )
2.  Documentation of findings and fixes to forward to dev or UI folx with access to modify HTML etc. 
3.  Post edit audit to ensure forward facing information can be accessed by all. 

##  💭 Who should be part of Advice Process on this issue (i.e., those in the community who should be consulted about this issue because they will have valuable input):
@AL0YSI0US 

## 🔗  The links and docs related to this issue:
+ https://wave.webaim.org/extension/ (ADA compliant WAVE browser extension)
+ https://tecommons.org/ (website)
+ https://token-engineering-commons.gitbook.io/tec-handbook/ (handbook)

**Comment by AL0YSI0US** [comment]
![website-audit](https://user-images.githubusercontent.com/75811965/132300412-7e878298-00d7-4754-b59e-60fc1c814407.PNG)

## 🔗 The links and docs related to this issue:
+ https://wave.webaim.org/extension/ (ADA compliant WAVE browser extension)
+ https://tecommons.org/ (website)

**Comment by AL0YSI0US** [comment]
![handbook-audit](https://user-images.githubusercontent.com/75811965/132300974-d6b67910-bb19-44db-a476-b21ccd70d325.PNG)
![contrast-errors-handbook](https://user-images.githubusercontent.com/75811965/132301000-4551e424-5007-4878-a5cd-9dd9c99beede.PNG)

**Comment by AL0YSI0US** [comment]
## ♿ Accessibility Support 👉🏾 [ MIRO BOARD ](https://miro.com/app/board/o9J_ltXbQjU=/?invite_link_id=274288856770)
> Created a Miro board to articulate each error found on the tokenengineeringcommons.org website.

-REVIEWED
✅ Mission
✅ Community
✅ Ecosystem
✅ Get Involved

![image](https://user-images.githubusercontent.com/75811965/135741066-11415205-c9bb-4ee1-9d2e-4c66ebb1a460.png)

## Errors
**Mission**
+ Empty Link (x2)
+ Missing Alternative Text (x1)
+ Missing form label (x1)

**Community**
+ Empty Link (x2)
+ Missing Alternative Text (x10)
+ Missing form label (x1)

**Ecosystem**
+ Empty Link (x2)
+ Missing Alternative Text (x1)
+ Missing form label (x1)

**Get Involved**
+ Empty Link (x2)
+ Missing Alternative Text (x1)
+ Very Low Contrast (x16)
+ Missing form label (x1)

**Comment by tamarandom** [comment]
@AL0YSI0US This is so thorough! We don't have dedicated front-end devs. We could really use some support there. I've tagged this "help needed".  Can anyone help us make our website accessibility compliant by making the changes @AL0YSI0US has thoroughly detailed!

**Comment by AL0YSI0US** [comment]
@tamarandom I had a Great call This past evening with @mcsmith89 & Dan, were itching to bring exactly what you're saying we don't have to the commons!

> The three of us gathered to talk desires and capacity around accessibility focused work in the commons. 
( 👇🏾 I took notes! )
+ [TEC Accessibility Jam Notes](https://docs.google.com/document/d/e/2PACX-1vRwKMxIcGx3a3-4UrS_XCaX2xRrc8HKQ2TtVJPL-NHAZ8XBT7KOhrShow6SVX4bQ2lvq_jPXIGGIB27/pub)

**Comment by mcsmith89** [comment]
The notes are looking great @AL0YSI0US! We're happy to help hop in and clear out some of the issues after the discussion yesterday - not sure how we can facilitate getting access to the main WordPress or if there is a repo and staging site to commit changes @tamarandom?

**Comment by tamarandom** [comment]
Tagging: @mcsmith89

**Comment by mcsmith89** [comment]
Hey Everyone! 

Meeting with Kristofer Thursday morning to figure out the structure and setup for website & changes. Will provide updates on here after that.

**Comment by kristoferlund** [comment]
Awesome with the detailed Miro board @AL0YSI0US! 

@mcsmith89, the repo is here: https://github.com/CommonsBuild/tec-web

What do you need from me to make the changes?

**Comment by mcsmith89** [comment]
Thanks! I am connecting with the dev team and will provide updates for anything needed
